project-base/app/src/Resources/definition/product/2.json

```diff
@@ -426,6 +426,16 @@
             "type": "keyword"
           }
         }
+      },
+      "hreflang_links": {
```

Comment:
I thought that you have to define `"type": "nested"` for this usage but I see we use it without that for breadcrumb already :shrug:

Reply:
According to documentation, the "nested" field should be used when it's mandatory to keep a relation between the items (hreflang and href). 
So I've added it in f333069895 and 19a411d1f2